sirepo/template/srw.py

```diff
@@ -1286,6 +1255,52 @@ def _enum_text(name, model, field):
     return ''
 
 
+def _export_rsopt_config(data):
+    v = _rsopt_jinja_context(data.models.exportRsOpt)
+    filename = 'rsOptExport.zip'
```

Comment:
Not sure if this temporary is necessary. Why not: p = pkio.py_path('rsOptExport.zip')?

Reply:
[075b377](https://github.com/radiasoft/sirepo/pull/4057/commits/075b377ec4e22e26dac2d429ee1d02791ddfeaf1)